src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -1,20 +1,14 @@
 package seedu.address.model.appointment;
 
-import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
-
-import java.util.Objects;
-
-import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.model.person.Person;
 
 /**
  * Represents an appointment with a date and time, a student, and a description.
  */
 public class Appointment {
-
-    private final DateTime dateTime;
-    private final Person student;
-    private final Description description;
+    private DateTime dateTime;
```

Comment:
Can take reference from `Person` class in the model.person package, it may be better to make the fields `final` for the objects to be **immutable.**